api/spec/requests/ingests_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+RSpec.describe "Projects API", type: :request do
+  include_context("authenticated request")
+  include_context("param helpers")
+
+  let!(:project) { FactoryBot.create(:project, draft: false) }
+  let(:project_id) { project.id }
+
+  let!(:attributes) {
+    {
+      source: markdown_source_params,
+      ingestionType: "epub"
+    }
+  }
+  let!(:valid_params) {
+    build_json_payload(attributes: attributes)
+  }
+
+  context "when a text ingest is triggered" do
+    let!(:path) { "/api/v1/projects/#{project_id}/ingest" }
```

Comment:
We shouldn't build our paths raw in request specs: `api_v1_projects_ingestions_path(project)` will get you a path.